lectures/sympy.md

```diff
@@ -0,0 +1,924 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.5
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+```{code-cell} ipython3
+!pip install sympy==1.12
```

Comment:
Is there a reason we need a specific version of sympy? Do updates often break and cause instability?

Reply:
Hi @mmcky,

This line was previously used to test whether a bug in SymPy has been fixed in the latest version of SymPy. The `pip` install has been removed because it is included in the base anaconda.